mantidproject/mantid
# Add legend font size option to Workbench settings

**Description of work.**
This PR adds an option to the workbench settings to specify the default legend font size.

**Report to:** Steve King


**To test:**

1. Open workbench settings
2. Go to the legend tab and set the font size (the default should be 8)
3. Plot some data, the legend font size should correspond to the value specified in the settings

Fixes #28851 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
